### PR TITLE
Auto ocp-33770 ocp-25598

### PR DESCRIPTION
**Pull request by xiuwang**
Passed locally.
@openshift/devexp-qe please help review, thanks

### CODE REVIEW

File: features/build/metrics.feature [hunk 1]
```diff
@@ -136,6 +136,88 @@ Feature: Builds and samples related metrics test
     Then the step should succeed
     And the output should contain:
       | cakephp-mysql-example-1  |
-      | sample-pipeline-1        |
```
Comment (liangxia) [inline features/build/metrics.feature:139]: The removal of this line is intended ?

File: features/build/metrics.feature [hunk 2]
```diff
@@ -136,6 +136,88 @@ Feature: Builds and samples related metrics test
...
+      | "repository":"devexp.registry-v1.qe.devcluster.openshift.com:5000/imagev1/test1".*value.*[0-9]* | 
+    """
+
+  # @author xiuwang@redhat.com
+  # @case_id OCP-25598
+  @admin
+  @destructive
+  Scenario: Monitoring, Alerting, and Degraded Status Reporting-Samples-operator 
+    When as admin I successfully merge patch resource "config.samples.operator.openshift.io/cluster" with:
+     | {"spec":{"samplesRegistry":"registry.unconnected.redhat.com"}} |
```
Comment (liangxia) [inline features/build/metrics.feature:181]: Indention issue.

File: features/build/metrics.feature (hunk 1 above)
Comment (xiuwang) [reply features/build/metrics.feature:139]: @liangxia It‘s intended, jenkins pipeline build has no build during time record.

Comment (liangxia) [reply features/build/metrics.feature:139]: Ack

**Pull request merged**